README.md

```diff
@@ -18,128 +18,21 @@ This is a placeholder for definitions of AsyncAPI specification extensions and b
-info:
-  title: My wonderful API
-  version: '12.4.2'
-  x-twitter: '@wonderapi'
-...
-```
-
-### HTTP Binding
-
-This is the example definition of an HTTP bindings extension.
-
-```yaml
-id: http
-type: bindings # Means it's a "bindings" type of extension.
-title: HTTP Binding Extension
-description: This object allows you to define HTTP-specific details in AsyncAPI.
-version: '0.1.0'
-author: Fran Mendez <fmvilas@gmail.com> (fmvilas.com)
-definitions:
-  - hooks: # JMESPath queries to indicate where this extension is allowed in the AsyncAPI document. In this case, it's allowed in all the operation (publish/subscribe) objects.
-      - 'channels.*.subscribe'
-      - 'channels.*.publish'
-    schema: # This is the schema that validates the extension content. It's in JSON Schema Draft 07.
-      type: object
-      required:
-        - version
-      properties:
-        version:
-          type: string
-          const: '0.1.0' # It's a good idea to specify what version of the extension are you using. It will facilitate things to tooling and will allow you to have different versions of the same extension in a single document.
-        response:
-          type: object
-          additionalProperties: false
-          properties:
-            headers:
-              type: object
-              additionalProperties: true
-              propertyNames:
-                type: string
-                pattern: '^[a-zA-Z0-9\\.\\-_]+$'
-        request:
-          type: object
-          additionalProperties: false
-          properties:
-            headers:
-              type: object
-              additionalProperties: true
-              propertyNames:
-                type: string
-                pattern: '^[a-zA-Z0-9\\.\\-_]+$'
-    examples:
-      - description: Usage for HTTP streaming APIs.
-        example:
-          http:
-            version: '0.1.0'
-            response:
-              headers:
-                Transfer-Encoding: chunked
-                Trailer: '\\r\\n'
-```
-
-#### Usage in an AsyncAPI document
-
-```yaml
-asyncapi: '2.0.0'
-...
-channels:
-  /tweets:
-    subscribe:
-      bindings:
-        http:
-          version: '0.1.0'
-          response:
-            headers:
-              Transfer-Encoding: chunked
-              Trailer: '\\r\\n'
-...
-```
-
-## Types of extensions
-
-Extensions can be of type `generic` or `bindings`.
-
-Generic extensions are those that are preceeded by `x-`, e.g., `x-twitter`, and they can be placed anywhere in the AsyncAPI document.
-
-Bindings extensions are those that live inside a `bindings` object and are related to protocol-specific functionality or characteristics. E.g., `http`, `kafka`, `amqp`, etc. As opposed to generic extensions, these can only be applied in a very limited set of places, namely, where the `bindings` is allowed, i.e., a channel object, an operation object or a message object.
+## List of Extensions
+- [Twitter Extension](./extensions/twitter/)
```

Comment:
Instead of having to maintain this list up to date all the time, I'd put here a link to the `extensions` folder in this repo. That would already act as a list itself and would be less maintenance overhead.

Reply:
@sambhavgupta0705 have a look at this comment

Reply:
ok so I am removing the list from here